spec/factories/likes.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :like do
+    association :user
+    association :tweet
```

Comment:
just a comment, here you can just do this:
```suggestion
    user
    tweet
```